linera-core/src/unit_tests/test_utils.rs

```diff
@@ -807,77 +801,84 @@ where
         balance: Amount,
     ) -> anyhow::Result<ChainClient<B::Storage>> {
         // Make sure the admin chain is initialized.
-        if self.genesis_storage_builder.accounts.is_empty() && index != 0 {
+        if self.admin_description.is_none() && index != 0 {
             Box::pin(self.add_root_chain(0, Amount::ZERO)).await?;
```

Comment:
This was for tests that don't explicitly initialize the admin chain. Maybe it isn't needed anymore? Either way, it doesn't make sense to just initialize an arbitrary root chain here.